graylog2-web-interface/src/components/event-definitions/hooks/useEventDefinitionConfigFromUrl.test.tsx

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import React from 'react';
+import { renderHook } from 'wrappedTestingLibrary/hooks';
+
+import useEventDefinitionConfigFromUrl from 'components/event-definitions/hooks/useEventDefinitionConfigFromUrl';
+import asMock from 'helpers/mocking/AsMock';
+import useQuery from 'routing/useQuery';
+import {
+  urlConfigWithAggString,
+  urlConfigWithFunctionAggString,
+  urlConfigWithoutAggString,
+} from 'fixtures/createEventDefinitionFromValue';
+
+jest.mock('routing/useQuery', () => jest.fn());
```

Comment:
In this case you can also just mock `useQuery` this way:
`jest.mock('routing/useQuery');`

Reply:
I will do changes in the follow-up PR, as I, probably, would have to rewrite some of the tests.